src/Sdk/NewsroomSubscriptions/types.ts

```diff
@@ -0,0 +1,38 @@
+import { Contact, UploadcareImage } from '../../types';
+
+export interface NewsroomSubscriptionCreateRequest {
+    email_address: string;
+    locale?: string;
+    url?: string | null;
```

Comment:
Do `null` and `undefined` produce the same behavior? 
If yes, then I'd avoid using nulls.
If not, then it'd be nice to leave a comment explaining the difference.
This comment applies to all other similar cases.

Reply:
Yes, parameter not sent is the same as it was `null`, if that is what you asked? I removed the nulls. 👍